src/api/character.ts

```
@@ -125,4 +126,11 @@ export class CharacterAPI implements ICharacterAPI {
   getFullName(character: Character) {
     return `${character.first_name} ${character.last_name}`;
   }
+
+  willAcceptReqruitment(character: Character) {
```

ESLint

[error] 'character' is defined but never used.

(@typescript-eslint/no-unused-vars)